src/main/java/io/vertx/core/Vertx.java

```diff
@@ -759,6 +759,12 @@ default <T> void executeBlocking(Handler<Promise<T>> blockingCodeHandler, Handle
    */
   WorkerExecutor createSharedWorkerExecutor(String name, int poolSize, long maxExecuteTime, TimeUnit maxExecuteTimeUnit);
 
+  /**
+   * @return whether virtual threads are available
+   */
+  @CacheReturn
+  boolean isVirtualThreadAvailable();
```

Comment:
I'd name this `areVIrtualThreadsAvailable()`

Reply:
maybe this should just be on VertxInternal, I introduced it for testing purposes

Reply:
I think it makes sense for the `Vertx` API to expose this (toolkit!).

Reply:
for now I prefer to keep it private and keep the API surface small, it actually tests that we can use reflection to build a virtual thread factory and not whether the runtime supports virtual threads